Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    payment_icons (1.4.4)
+    payment_icons (1.4.5)
```

Comment:
Can you please revert this file? I will be the one bumping the version.

Reply:
Done